constants/types.ts

```diff
@@ -17,7 +17,21 @@ export interface Package {
   reason_for_deactivation?: string | null;
   deactivated_at?: string | null;
 }
-
+export interface EditPackageProps {
```

Comment:
this is not named properly. please follow the naming convention already used on this file